.idea/codeStyles/Project.xml

```diff
@@ -157,4 +159,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

Reply:
@Scarlet0103 this is still pending.